corehq/form_processor/management/commands/run_sql.py

```diff
@@ -497,7 +497,15 @@ def meta_fields(meta, with_variable_fields=True):
 """)
 
 
+# see https://github.com/dimagi/commcare-hq/pull/22932
+blobmeta_partial_key = """
+CREATE UNIQUE INDEX CONCURRENTLY IF NOT EXISTS blobs_blobm_key_14b5b7_partial
+ON public.blobs_blobmeta (key) WHERE deleted_on IS NULL;
```

Comment:
you meant to keep it "IS NULL"?

Reply:
Yes, it excludes deleted rows from the index.

Reply:
a deleted row would have `deleted_on` set and hence this should be `NOT NULL`?
fine to ignore it if its not relevant anymore